terraform/modules/iam/main.tf

```diff
@@ -0,0 +1,66 @@
+##########################################################
+# IAM
+#
+# Spree application user
+##########################################################
+resource "aws_iam_user" "spree" {
+  name = "spree-app-user"
+}
+
+resource "aws_iam_access_key" "spree" {
+  user = aws_iam_user.spree.name
+}
+
+# Create System Parameters as we need these to inject as secrets into ECS
+resource "aws_ssm_parameter" "aws_access_key_id" {
+  name      = "/bat/${lower(var.environment)}-aws-access-key-id"
+  type      = "SecureString"
+  value     = aws_iam_access_key.spree.id
+  overwrite = true
+}
+
+resource "aws_ssm_parameter" "aws_secret_access_key" {
+  name      = "/bat/${lower(var.environment)}-aws-secret-access-key"
+  type      = "SecureString"
+  value     = aws_iam_access_key.spree.secret
+  overwrite = true
```

Comment:
Need to encrypt these two using the new CMK - see example in previous PR: https://github.com/Crown-Commercial-Service/ccs-scale-infra-services-fat/pull/170/files#diff-414d0fcf4cadd0901a7bc0ec77e1bb1d265d51d2ee55f84da485cbeef55d677cR93-R102

Reply:
Great point - forgot to update after that change - will do so now 👍